src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/AddUserAccessForm.tsx

```diff
@@ -138,6 +137,7 @@ export const AddUserAccessForm = ({
               const trimmed = e.target.value.trim();
               debouncedSetSearchedStr(trimmed);
             }}
+            text={intl.formatMessage(messages.searchInfos)}
```

Comment:
I find awkward to have this text help under the search field.
I think we should keep it right after the title.